synapse/storage/databases/main/user_directory.py

```diff
@@ -40,12 +40,10 @@
 
 logger = logging.getLogger(__name__)
 
-
```

Comment:
I'm not sure how these crept in. Possibly I accidentally formatted in pycharm (ctrl-alt-l) and `black` was happy with the change?